### DISCUSSION TITLE
Cleaning up dynamic partitions

### DISCUSSION THREAD

**Discussion by cody-scott** [dagster-io/dagster#20200] [Q&A]
Hi, 

Is there a better way to clean up removed dynamic partitions then modifying the db? I see a number that were in older assets that are no longer used, and wouldn't mind cleaning those up. Before i head off editing the db, wanted to confirm if there was a better way.

I cant see any other tightly linked tables to this, so my first guess is it won't be a huge issue to drop those rows, but a quick confirm would be great.

**Comment by garethbrickman** [comment]
hey Cody, unfortunately there isn't a way to do this within the UI. There is a feature request in #14026  for tracking that, and in its comments are examples of how to do so programmatically
